javascript-forms/form-values.js

```diff
@@ -0,0 +1,15 @@
+var form = document.querySelector('form');
+
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  console.log('Form Submit');
+
+  var formElements = {
+    name: document.forms[0].elements.name.value,
+    email: document.forms[0].elements.email.value,
+    message: document.forms[0].elements.message.value
+  };
+  console.log('Form Elements: ', formElements);
+  document.forms[0].reset();
```

Comment:
Instead of using `document.forms[0]`, you can just use `form` since you already queried for that element and saved it into a variable